src/components/common/DefaultIconText.tsx

```diff
@@ -14,38 +14,24 @@ const DefaultIconText: FC<DefaultIconTextProps> = ({
     item,
     defaultCardImageIcon
 }) => {
-    if (item.CollectionType) {
-        return (
-            <Icon
-                className='cardImageIcon'
-                sx={{ color: 'inherit', fontSize: '5em' }}
-                aria-hidden='true'
-            >
-                {imageHelper.getLibraryIcon(item.CollectionType)}
-            </Icon>
-        );
+    let icon;
+
+    if (item.Type === BaseItemKind.CollectionFolder || item.CollectionType) {
```

Comment:
Recordings item should be a folder instead of quiz. The Recordings item type value is `CollectionFolder` but the `CollectionType` is undefined

Reply:
It is also a type of mixed library so I think using the same icon is fine :man_shrugging: